R/autoplot.pop_data.R

```diff
@@ -102,11 +121,34 @@ density_plot <- function(
       ggplot2::labs(fill = strata)
   }
   if (log) {
+    min_nonzero_val <-
+      object %>%
+      filter(object %>% get_value_var() > 0) %>%
```

Comment:
`get_value_var()` should be `get_value()`; that's why we were getting 0s in line 128.